### DISCUSSION TITLE
[Feature] Web - When Searching, Enable multi-select with Shift

### DISCUSSION THREAD

**Discussion by maxshcherbina** [immich-app/immich#7501] [Feature Request]
### The feature

When looking for photos in search on web, I cannot Shift+Click photos to select multiple photos. 

This feature exists in the main application window, and I feel like it should also exist on the search screen.

Itd be great if the implementation of #1659 was carried over to the search window!

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #7272

**Reply by maxshcherbina** [reply]
Sorry - thank you! Needed to search longer.

Go to #9159  for further discussion on this request.
